26/06/23 (16:37:52) | ERROR: Exception while getting cpu frequency!
26/06/23 (16:37:52) | ERROR: [Errno 2] No such file or directory (originated from sysctl(HW_CPU_FREQ))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3080, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
                                        ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.11/site-packages/psutil/__init__.py", line 1864, in cpu_freq
    ret = _psplatform.cpu_freq()
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.11/site-packages/psutil/_psosx.py", line 179, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
                       ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctl(HW_CPU_FREQ))
25/07/23 (13:26:04) | ERROR: Exception while getting cpu frequency!
25/07/23 (13:26:04) | ERROR: [Errno 2] No such file or directory (originated from sysctl(HW_CPU_FREQ))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3080, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
                                        ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.11/site-packages/psutil/__init__.py", line 1864, in cpu_freq
    ret = _psplatform.cpu_freq()
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/bpytop/1.0.68/libexec/lib/python3.11/site-packages/psutil/_psosx.py", line 179, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
                       ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctl(HW_CPU_FREQ))
